website/public/js/controllers/authCtrl.js

```diff
@@ -29,7 +29,7 @@ angular.module('habitrpg')
         if (status === 0) {
           $window.alert(window.env.t('noReachServer'));
         } else if (!!data && !!data.err) {
-          $window.alert(data.err);
+          $window.alert(data.data.message);
```

Comment:
Errors return {error, success, message} as response, data is only used for successful responses ( {data, success} )